platform-grpc-service-framework/src/main/java/org/hypertrace/core/serviceframework/grpc/GrpcPlatformServiceContainer.java

```diff
@@ -222,12 +236,12 @@ protected abstract GrpcServiceContainerEnvironment buildContainerEnvironment(
   private ServerBuilder<?> initializeBuilder(GrpcPlatformServerDefinition serverDefinition) {
     ServerBuilder<?> builder = ServerBuilder.forPort(serverDefinition.getPort());
 
-    // add micrometer-grpc interceptor to collect server metrics.
-    builder.intercept(new MetricCollectingServerInterceptor(PlatformMetricsRegistry.getMeterRegistry()));
-
     if (serverDefinition.getMaxInboundMessageSize() > 0) {
       builder.maxInboundMessageSize(serverDefinition.getMaxInboundMessageSize());
     }
+    // add micrometer-grpc interceptor to collect server metrics.
+    builder.intercept(
```

Comment:
Any reference of what all metrics we'll get by this?

Reply:
We're adding both server and client interceptors which will get us call counts and response times by:
[host service, called service, called method, status code]

Reply:
Some examples Laxman had shared:

Server:
```

grpc_server_processing_duration_seconds_count{method="Check",methodType="UNARY",service="grpc.health.v1.Health",statusCode="OK",} 64.0
grpc_server_processing_duration_seconds_sum{method="Check",methodType="UNARY",service="grpc.health.v1.Health",statusCode="OK",} 0.107004252
grpc_server_processing_duration_seconds_max{method="Check",methodType="UNARY",service="grpc.health.v1.Health",statusCode="OK",} 0.002033641

grpc_server_processing_duration_seconds_count{method="execute",methodType="SERVER_STREAMING",service="org.hypertrace.core.query.service.QueryService",statusCode="OK",} 20.0
grpc_server_processing_duration_seconds_sum{method="execute",methodType="SERVER_STREAMING",service="org.hypertrace.core.query.service.QueryService",statusCode="OK",} 100.835870121
grpc_server_processing_duration_seconds_max{method="execute",methodType="SERVER_STREAMING",service="org.hypertrace.core.query.service.QueryService",statusCode="OK",} 7.925791017
```

Client:
```
grpc_client_requests_sent_messages_total{method="Check",methodType="UNARY",service="grpc.health.v1.Health",} 64.0
grpc_client_requests_sent_messages_total{method="findAttributes",methodType="SERVER_STREAMING",service="org.hypertrace.core.attribute.service.v1.AttributeService",} 1.0
```